inference-engine/src/transformations/include/transformations/depth_to_space_fusion.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <memory>
+
+#include <ie_api.h>
+
+#include <ngraph/pass/graph_rewrite.hpp>
+#include "transformations/utils/pass_param.hpp"
+
+namespace ngraph {
+namespace pass {
+
+    class INFERENCE_ENGINE_API_CLASS(DepthToSpaceFusion);
+
+}  // namespace pass
+}  // namespace ngraph
+
+class ngraph::pass::DepthToSpaceFusion: public ngraph::pass::GraphRewrite, public ngraph::pass::PassParam {
```

Comment:
Add description.
Look at this example: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/transformations/include/transformations/convert_opset1_to_legacy/reshape_fully_connected.hpp

Reply:
the description was added.